packages/bpk-component-scrollable-calendar/src/BpkScrollableCalendarDate.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import PropTypes from 'prop-types';
+import React, { PureComponent } from 'react';
+import { BpkCalendarDate, CustomPropTypes } from 'bpk-component-calendar';
+
+class BpkScrollableCalendarDate extends PureComponent {
+  render() {
+    const { date, isOutside, ...rest } = this.props;
+
+    // Returning empty span when isOutside is true ensures only focused month displays in grid
+    return !isOutside ? <BpkCalendarDate date={date} {...rest} /> : <span />;
+  }
+}
+
+BpkScrollableCalendarDate.propTypes = {
+  // Required
+  date: PropTypes.instanceOf(Date).isRequired,
+  // Optional
+  isBlocked: PropTypes.bool,
+  isFocused: PropTypes.bool,
+  isOutside: PropTypes.bool,
+  isSelected: PropTypes.bool,
+  isToday: PropTypes.bool,
+  modifiers: CustomPropTypes.DateModifiers,
+  onClick: PropTypes.func,
+  onDateKeyDown: PropTypes.func,
+  preventKeyboardFocus: PropTypes.bool,
```

Comment:
Shouldn't we spread all `CustomPropTypes` here?

Reply:
Could you explain what you mean? I am using the same the same props as BpkCalendarDate